### DISCUSSION TITLE
Why `Label` ?

### DISCUSSION THREAD

**Discussion by sudoaugustin** [radix-ui/primitives#1860] [Help]
### Why should we use radix's label component over a native label with the `user-select: none`  CSS property?

**Comment by joaom00** [comment]
The main difference I see is that the text selection is _only_ prevented on double clicking, you can still select label normally

**Comment by benoitgrelard** [comment]
At this point in time, it doesn't provide much more than a native `label`.
This is more of a philosophical choice, in the sense that we want to provide APIs that we can add too. So although it may not do more now, it could in the future so it would be good practice to use the primitive.

Additionally, note that `user-select: none` won't result in the same behaviour as it will prevent users selecting the label text which can be an accessibility issue. Ours prevents the double-click selection behaviour (as it can happen unintentionally when clicking a label to activate its control) but doesn't prevent users from selecting the text normally by click-drag.

**Reply by joaom00** [reply]
The example in the docs set `user-select: none` in styles, was it to be there?

**Reply by benoitgrelard** [reply]
Ah thanks for pointing that out, not it shouldn't be there and isn't necessary.

**Reply by benoitgrelard** [reply]
https://github.com/radix-ui/website/pull/556

**Reply by sudoaugustin** [reply]
Thx guys 🙏
